lnbits/wallets/clightning.py

```diff
@@ -84,9 +86,17 @@ async def create_invoice(
             error_message = f"lightningd '{exc.method}' failed with '{exc.error}'."
             return InvoiceResponse(False, label, None, error_message)
 
-    async def pay_invoice(self, bolt11: str) -> PaymentResponse:
+    async def pay_invoice(self, bolt11: str, fee_limit_msat: int) -> PaymentResponse:
+        invoice = lnbits_bolt11.decode(bolt11)
+        fee_limit_percent = (invoice.amount_msat // fee_limit_msat) // 100
+
+        payload = {
+            "bolt11" : bolt11,
+            "maxfeepercent" : fee_limit_percent
+        }
+        
         try:
-            r = self.ln.pay(bolt11)
+            r = self.ln.call("pay", payload)
```

Comment:
I can't test this :) Did you?

Reply:
I tested it only on Polar with c-lightning v0.10.0 and pylightning 0.0.7.3. I've tested it again right now with your inverted fee calc.